lib/python/ray/worker.py

```diff
@@ -686,31 +686,27 @@ def process_task(task): # wrapping these lines in a function should cause the lo
       # above so that changes made to their state do not affect other tasks.
       ray.reusables._reinitialize()
   while True:
-    (task, function, reusable_variable) = ray.lib.wait_for_next_message(worker.handle)
+    item_type, item = ray.lib.wait_for_next_message(worker.handle)
```

Comment:
why don't you call them "command, args = wait_for_next_message(...)" and then

command = "die", "register_function", "register_reusable_var", "exec_task"

Item is a very generic word.